stdlib/dynArray.mli

```diff
@@ -0,0 +1,52 @@
+(** Dynamic Arrays.
+
+   This module implements amortized time complexity dynamic arrays.
+ *)
+
+type 'a dynArray
+(** The type of dynamic arrays containing elements of type ['a]. *)
+
+exception OutOfBounds
+(** Raised when an index is out of the array. *)
```

Comment:
Why not use `Invalid_argument` like Array does?

Reply:
Yeah, that’s obviously better. I am going to change it.